[2025-09-22T00:44:36.581+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: processamento_silver.geolocation_to_silver manual__2025-09-22T00:40:46.287154+00:00 [queued]>
[2025-09-22T00:44:36.595+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: processamento_silver.geolocation_to_silver manual__2025-09-22T00:40:46.287154+00:00 [queued]>
[2025-09-22T00:44:36.596+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-09-22T00:44:36.653+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): geolocation_to_silver> on 2025-09-22 00:40:46.287154+00:00
[2025-09-22T00:44:36.662+0000] {standard_task_runner.py:60} INFO - Started process 1090 to run task
[2025-09-22T00:44:36.673+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'processamento_silver', 'geolocation_to_silver', 'manual__2025-09-22T00:40:46.287154+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/processamento_silver.py', '--cfg-path', '/tmp/tmp7bpkwoip']
[2025-09-22T00:44:36.679+0000] {standard_task_runner.py:88} INFO - Job 113: Subtask geolocation_to_silver
[2025-09-22T00:44:36.829+0000] {task_command.py:423} INFO - Running <TaskInstance: processamento_silver.geolocation_to_silver manual__2025-09-22T00:40:46.287154+00:00 [running]> on host e833383cbb8e
[2025-09-22T00:44:37.178+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='processamento_silver' AIRFLOW_CTX_TASK_ID='geolocation_to_silver' AIRFLOW_CTX_EXECUTION_DATE='2025-09-22T00:40:46.287154+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-22T00:40:46.287154+00:00'
[2025-09-22T00:44:37.190+0000] {sql.py:276} INFO - Executing: CREATE SCHEMA IF NOT EXISTS silver;

DROP TABLE IF EXISTS silver.olist_geolocation_dataset;

CREATE TABLE silver.olist_geolocation_dataset AS
SELECT
    regexp_replace(geolocation_zip_code_prefix, '[^0-9]', '', 'g') AS geolocation_zip_code_prefix,
    CAST(geolocation_lat AS NUMERIC(18,6)) AS geolocation_lat,
    CAST(geolocation_lng AS NUMERIC(18,6)) AS geolocation_lng,
    lower(trim(geolocation_city)) AS geolocation_city,
    upper(trim(geolocation_state)) AS geolocation_state
FROM bronze.olist_geolocation_dataset;
[2025-09-22T00:44:37.223+0000] {base.py:83} INFO - Using connection ID '***_db' for task execution.
[2025-09-22T00:44:37.247+0000] {base.py:83} INFO - Using connection ID '***_db' for task execution.
[2025-09-22T00:44:37.258+0000] {sql.py:450} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS silver;

DROP TABLE IF EXISTS silver.olist_geolocation_dataset;

CREATE TABLE silver.olist_geolocation_dataset AS
SELECT
    regexp_replace(geolocation_zip_code_prefix, '[^0-9]', '', 'g') AS geolocation_zip_code_prefix,
    CAST(geolocation_lat AS NUMERIC(18,6)) AS geolocation_lat,
    CAST(geolocation_lng AS NUMERIC(18,6)) AS geolocation_lng,
    lower(trim(geolocation_city)) AS geolocation_city,
    upper(trim(geolocation_state)) AS geolocation_state
FROM bronze.olist_geolocation_dataset;, parameters: None
[2025-09-22T00:44:37.260+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 282, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 398, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 455, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedFunction: function regexp_replace(bigint, unknown, unknown, unknown) does not exist
LINE 7:     regexp_replace(geolocation_zip_code_prefix, '[^0-9]', ''...
            ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.

[2025-09-22T00:44:37.274+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=processamento_silver, task_id=geolocation_to_silver, execution_date=20250922T004046, start_date=20250922T004436, end_date=20250922T004437
[2025-09-22T00:44:37.450+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 113 for task geolocation_to_silver (function regexp_replace(bigint, unknown, unknown, unknown) does not exist
LINE 7:     regexp_replace(geolocation_zip_code_prefix, '[^0-9]', ''...
            ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
; 1090)
[2025-09-22T00:44:37.476+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-09-22T00:44:37.603+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
